### PR TITLE
Configure ESLint to handle TS

### PR DESCRIPTION
**Pull request by Brailor**

### CODE REVIEW

**Review by matyasf — APPROVED** [review]
nice work!

File: packages/ui-avatar/.eslintignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /node_modules/**
 /lib/**
 /es/**
+/types/**
+/types/**
```
Comment (ke1k0) [inline packages/ui-avatar/.eslintignore:5]: duplication :)

**Review by ke1k0 — APPROVED** [review]
there is one small duplication, otherwise works nice :)

File: packages/ui-avatar/.eslintignore (hunk 1 above)
Comment (Brailor) [reply packages/ui-avatar/.eslintignore]: You are right, removed

**Pull request merged**